src/Client/ClientFactory.php

```diff
@@ -31,9 +31,9 @@ public function createFromApiCredentials(): AkeneoPimClientInterface
         return $this->authenticateByPassword($apiConfiguration);
     }
 
-    public function authenticateByPassword(ApiConfiguration $apiConfiguration): AkeneoPimClientInterface
+    public function authenticateByPassword(ApiConfiguration $apiConfiguration): AkeneoPimEnterpriseClientInterface
     {
-        $client = new AkeneoPimClientBuilder($apiConfiguration->getBaseUrl() ?? '');
+        $client = new AkeneoPimEnterpriseClientBuilder($apiConfiguration->getBaseUrl() ?? '');
```

Comment:
If we don't have the enterprise version, doesn't that introduce a bug?

Reply:
no incidence, we just have to make sure to not call enterprise endpoints

Reply:
AkeneoPimEnterpriseClient fallback to AkeneoPimClient to called classic endpoints.